experimental-statement.md

```diff
@@ -0,0 +1,21 @@
+# Experimental Statement
```

Comment:
I think it will be better to link out to the equinix-labs/equinix-labs copy, less maintenance downstream. This text is something we will likely be revisiting.

Same sentiment expressed here: https://github.com/equinix-labs/metal-runner-action/pull/4/files#r1250979819